stable/openstack-base/README.md

```diff
@@ -247,6 +247,11 @@ To print your credentials:
     source openrc
     env | grep OS_
 
+Enable a remote access protocol such as `novnc` (or `spice`) if you want to
```

Comment:
This change should be done to the `development/` READMEs as well, for focal-victoria and above, thanks!